internal/backuprestore/common.go

```diff
@@ -103,6 +105,14 @@ type BRStatusError struct {
 	ErrMessage string
 }
 
+type GVRNamespaceName struct {
```

Comment:
You mean `NamespacedName` yeah? Can we have something a little generic e.g `ObjMetadata`?